api/tests/integration/environments/test_environments.py

```diff
@@ -0,0 +1,15 @@
+from django.urls import reverse
+from rest_framework import status
+
+
+def test_clone_environment_returns_200(
+    admin_client, project, environment, environment_api_key
+):
+    # Given
+    url = reverse("api-v1:environments:environment-clone", args=[environment_api_key])
+    # When
+    res = admin_client.post(url, {"name": "clone-env"})
+
+    # Then
+    assert res.status_code == status.HTTP_200_OK
+    assert res.json()["name"] == "clone-env"
```

Comment:
Very picky but we should pull "clone-env" out as a variable so it makes the test easier to extend / refactor if we need to.